features/standards/mag/structure/02_headings.feature

```diff
@@ -280,3 +280,62 @@ Feature: Headings
       """
     When I test the "Structure: Headings: Content must follow headings" standard
     Then it passes
+
+  @html @automated
```

Comment:
These scenarios look like they are indented inconsistently with the rest of the file

Reply:
Think it's the same now?